### ISSUE TITLE
Prediction too far off

### ISSUE THREAD

**Issue by SpirikleOfficial** [prabhakar-sivanesan/Custom-keypoint-detection#6]
I am attaching the colab file which was used for training my document edge detection model. The colab file was cloned from this github repo. Some steps like tfrecord generation and tensorflow github clone is not shown in the colab because it was done already and I had it commented out when I reran the notebook. Please remember the total loss value was 0.08 which should be good for making predictions.

Main Concern:
Why the predictions are to far off when the loss was low? I have followed every step and can't pinpoint to the reason.

Colab file:
[my_model.zip](https://github.com/prabhakar-sivanesan/Custom-keypoint-detection/files/14922789/my_model.zip)

Video for the same:

https://github.com/prabhakar-sivanesan/Custom-keypoint-detection/assets/109790467/5019e989-ab0f-4acc-8053-123b9b743765
